.github/workflows/development_infrastructure.yml

```diff
@@ -757,6 +786,64 @@ jobs:
     #       fi
   # END Redis setup
 
+  # START ClamAV setup
+  clamav:
+    name: ClamAV Setup
+    needs: [set-environment, vnet, private-dns]
+    environment:
+      name: ${{ needs.set-environment.outputs.environment }}
+    runs-on: [self-hosted, linux, deployment]
+    steps:
+      - uses: azure/login@v1
+        with:
+          creds: ${{ env.credentials }}
+
+      - name: Defaults
+        uses: azure/cli@v1.0.0
+        with:
+          inlineScript: |
+            az configure --defaults location=uksouth
+            az configure --defaults group=${{secrets.RESOURCE_GROUP }}
+
+      # Need to set up log-analytics and use existing workspace resource here for managed environment and application insights
+      # - name: Workspace Setup
+      #   uses: azure/cli@v1.0.0
+      #   with:
+      #     inlineScript: |
+      #       az monitor log-analytics workspace create --workspace-name ${{ secrets.RESOURCE_GROUP}}-Logs-Workspace
+      
+      - name: Managed Environment
+        uses: azure/cli@v1.0.0
+        with:
+          inlineScript: |
+            acaClamAvSubnetId=$(az vnet show --name tfs-${{ env.app_service_plan }}-vnet --query "[?name=='${{ env.environment }}-clamav'].id" --output tsv)
```

Comment:
Ah I guess we should have tested this - should we just test this against the deployed BICEP clamav to see if it returns a sensible looking id?

Reply:
Good idea, this one didn't work but tried another command which has worked